### ISSUE TITLE
Detection errors for bicycle on CityScapes using the provided pre-trained models

### ISSUE THREAD

**Issue by yizhou-wang** [open-mmlab/mmdetection#1746]
I am using the provided pre-trained models (FRCNN & MRCNN) on CityScapes dataset: https://github.com/open-mmlab/mmdetection/tree/master/configs/cityscapes. However, the inferences seems to be wrong for most bicycles (one example is shown below). The bicycles are detected as trucks for most of the times. Did anyone meet the same problem? Is there any errors while training the models on CityScapes? Thanks!

![_screenshot_03 12 2019](https://user-images.githubusercontent.com/24263207/70117270-e29aa300-1619-11ea-877e-f619214c4db9.png)

**Comment by hellock** [comment]
Maybe the label mapping is incorrect. @yhcao6 You could have a look.

**Comment by yhcao6** [comment]
Thanks for reporting bug. Please check pr #1758, the original CLASS order is wrong. You need download model again.

**Comment by yl-precious** [comment]
> Thanks for reporting bug. Please check pr #1758, the original CLASS order is wrong. You need download model again.

Hello, I download the pretrained model to validate the performance on the val set. But I got a terrible results as follows:

` Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.042`
 `Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.084`
 `Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.037`
`Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.067`
` Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.055`
` Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.052`
 `Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.035`
 `Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.069`
` Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.071`
` Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.071`
` Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.071`
` Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.071`

The json files are generated following [https://github.com/facebookresearch/maskrcnn-benchmark/tree/master/maskrcnn_benchmark/data](url). 
 I visualized the results but found it not so bad yet. Is the problem when calculating the AP?

**Comment by yl-precious** [comment]
> > Thanks for reporting bug. Please check pr #1758, the original CLASS order is wrong. You need download model again.
> 
> Hello, I download the pretrained model to validate the performance on the val set. But I got a terrible results as follows:
> 
> ` Average Precision (AP) @[ IoU=0.50:0.95 | area= all | maxDets=100 ] = 0.042`
> `Average Precision (AP) @[ IoU=0.50 | area= all | maxDets=100 ] = 0.084`
> `Average Precision (AP) @[ IoU=0.75 | area= all | maxDets=100 ] = 0.037`
> `Average Precision (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.067`
> ` Average Precision (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.055`
> ` Average Precision (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.052`
> `Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets= 1 ] = 0.035`
> `Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets= 10 ] = 0.069`
> ` Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets=100 ] = 0.071`
> ` Average Recall (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.071`
> ` Average Recall (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.071`
> ` Average Recall (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.071`
> 
> The json files are generated following [https://github.com/facebookresearch/maskrcnn-benchmark/tree/master/maskrcnn_benchmark/data](url).
> I visualized the results but found it not so bad yet. Is the problem when calculating the AP?

I solved it, thank you
